staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -852,6 +848,24 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	return &apiResource, nil
 }
 
+// getOpenAPISchema builds the openapi schema for a single resource model to be given to each handler. It will
+// return nil if the apiserver doesn't have openapi enabled, or if the specific path should be ignored by openapi.
+func (a *APIInstaller) getOpenAPISchema(rootPath, resource string, kind schema.GroupVersionKind, sampleObject interface{}) (openapiproto.Schema, error) {
+	path := gpath.Join(rootPath, resource)
+	if a.group.OpenAPIConfig == nil {
+		return nil, nil
+	}
+	pathsToIgnore := openapiutil.NewTrie(a.group.OpenAPIConfig.IgnorePrefixes)
```

Comment:
Creating a Tri and use it only once look like a waste of space.

Reply:
I think it would be garbage collected once this installer finishes, right?

Reply:
If the Trie is < 1GB I think it's fine :)

Reply:
Even if not, is there a better way to do this? What else is the CPU going to do with those cycles anyway? :)

Reply:
It might make sense to build this only once, when APIInstaller is constructed. But I wouldn't bother with that unless a profiler shows this is a hot spot.

Reply:
I'm not sure we even use OpenAPIConfig.IgnorePrefixes in the kube apiserver, since things were working fine in kube apiserver without this PR. I think maybe some extension apiservers might use it, but I'm not sure how extensively.